### DISCUSSION TITLE
Relative sqlite connection url

### DISCUSSION THREAD

**Discussion by hpohlmeyer** [prisma/prisma#9642] [Q&A] [answered]
Hi,
I am not sure if this is a bug or intended behavior.

I have a simple setup based on sqlite. My schema looks like this:

```ts
// This is your Prisma schema file,
// learn more about it in the docs: https://pris.ly/d/prisma-schema

datasource db {
  provider = "sqlite"
  url      = "file:./dev.db"
}

generator client {
  provider = "prisma-client-js"
}

model User {
  id    Int     @default(autoincrement()) @id
  email String  @unique
  name  String?
}
```

It is closely following https://www.prisma.io/docs/concepts/database-connectors/sqlite

Initializing the db with `prisma migrate dev --name init` works fine and initilizes my database in `prisma/dev.db`.
Generating the client with `prisma generate` works fine as well, but using the client does not work. Every prisma client method fails with an error that says the table does not exist in the database.

After some digging I have realized, that a call to a prisma method creates an empty `dev.db` file in the prisma client folder (`node_modules/.prisma/client/dev.db`).

I would have expected, that the root path for the relative url would be consistent, otherwise there is no point in using a relative url, or am I using it wrong?

**Comment by janpio** [comment, marked as answer]
Can you reproduce this? If so, can you please open an issue with your discussion text? That is definitely _not_ expected and should not happen. Please make sure to fill the issue template so we have all the information, and optimally also provide a minimal reproduction that you know works so we can easily reproduce this. Thanks!

**Reply by hpohlmeyer** [reply]
I have tracked it down further to be rooted in the hoisting behavior of npm workspaces. I  will create an issue for it. Thank you for your feedback!

**Reply by hpohlmeyer** [reply]
Here it is: https://github.com/prisma/prisma/issues/9649

**Reply by janpio** [reply]
Great work. Thanks!

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
